src/test/java/org/elasticsearch/search/suggest/phrase/NoisyChannelSpellCheckerTests.java

```diff
@@ -279,8 +279,11 @@ protected TokenStreamComponents createComponents(String fieldName, Reader reader
         assertThat(corrections.length, equalTo(1));
         assertThat(corrections[0].join(new BytesRef(" ")).utf8ToString(), equalTo("xorr the god jewel"));
 
+        corrections = suggester.getCorrections(wrapper, new BytesRef("Quazar II"), generator, 1, 1, ir, "body", wordScorer, 1, 2).corrections;
```

Comment:
And some kind of comment here about how this name hits a pre filter that doesn't make any changes.